modules/uptimes.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+ SCRIPTPATH=$(realpath "$0")
+ SCRIPTFOLDER=$(dirname "$SCRIPTPATH")
+
+function uptimes(){
+if [[ ! -f "$SCRIPTFOLDER/templates/up_time" ]]
+then
+uptime -s > "$SCRIPTFOLDER"/templates/up_time 
+up=$(uptime -s)
+echo "System is up since $up"
+else
+last_up_time=$(cat "$SCRIPTFOLDER"/templates/up_time)
+curr_up_time=$(uptime -s)
+if [[ "$last_up_time" != "$curr_up_time" ]]
+then
+uptime -s > "$SCRIPTFOLDER"/templates/up_time
+echo "Down Time was between $last_up_time and $curr_up_time"
+echo "System has been up since $curr_up_time"
+else
+up=$(uptime -s)
+echo "System is up since $up"
+fi
+fi 
+}
```

Comment:
Additionally, we'll need another function within this shell in order for our help shell to function properly. 
Please add under your function an `uptimes_help` function with "Usage" and an "Example" I'll paste a copy of `upgrade` for a reference:
```
function upgrade_help {
  echo ""
  echo "Usage: $(basename "$0") upgrade [-f] [tag] [--check]"
  echo ""
  echo "Upgrades $(basename "$0") package using npm"
  echo ""
  echo "Example:"
  echo " $(basename "$0") upgrade"
  echo "    This will upgrade the $(basename "$0") package using npm and will try to install the latest version of $(basename "$0") running on your system"
  echo "    If the latest version if installed it will not try to reinstall it"
  echo ""
  echo " $(basename "$0") upgrade -f"
  echo "    This will do the same as upgrade, but the version will not be checked. Meaning that this will force the installation of the latest version"
  echo ""
  echo " $(basename "$0") upgrade tag"
  echo "    This will upgrade the $(basename "$0") package to the version with the specified tag"
  echo ""
  echo " $(basename "$0") upgrade --check"
  echo "    checks if there is a new version of the package, outputs false if there isnt, outputs true + version if there is"
  echo ""
}
```
Once added we can merge your work in. 😄

Reply:
thanks for testing and reviewing will apply the changes now.